person.rb

```diff
@@ -1,22 +1,36 @@
-class Person
-  attr_reader @id
+require_relative 'nameable'
+require_relative 'rental'
 
+class Person < Nameable
+  attr_reader @id, @rental
```

Comment:
## [OPTIONAL] 
* I strongly suggest that you also add a setter for @rentals ( ie attr_accessor @rentals) as you have done in other classes with the add_rental method